docs/network/monitoring.md

```diff
@@ -0,0 +1,332 @@
+section, to add the grafana package repository.
+
+Install Loki on the graphana server and Promtail on the EVM server.
+
+Once both are installed, you need to add the scraping job to the Promtail 
+configuration file (e.g. `/etc/promtail/config.yaml`) and specify the target 
+Loki server.
+
+```yaml
+clients:
+- url: <LOKI_ENDPOINT>
+```
+
+You must also configure a parsing pipeline for the logs.
+Below is an example of a pipeline suitable for the daily logs. It uses `grep`
+to extract the event's name from the log, to make searching the logs easier. 
+Alternatively, we present [later](#adding-log-sinks) a method to produce logs in a json format, and the 
+appropriate Promtail pipeline.
+
+```yaml
+scrape_configs:
+- job_name: "evm-node-log-exporter" 
+  pipeline_stages:
+    - regex:
+        expression: "^(?P<time>\\S+)\\s*\\[(?P<event>\\S+)\\](?P<msg>.*)$"
+    - timestamp:
+        source: time
+        format: RFC3339
+    - labels:
+        event:
+  static_configs:
+  - targets:
+      - 127.0.0.1
+    labels:
+      __path__: <EVM_NODE_DATA_DIR>/daily_logs/*.log
+
+```
+
+Replace `<EVM_NODE_DATA_DIR>` with the path to the EVM node data dir, and
+`<LOKI_ENDPOINT>` with the Loki endpoint, most probably:
+
+```
+http://<LOKI_SERVER_ADDRESS>:3100/loki/api/v1/push
+```
+
+so if Loki is installed on the same server:
+
+```
+http://localhost:3100/loki/api/v1/push
+```
+
+Make sure that the Promtail daemon is able to access the log files. If it's 
+installed as a service, an easy way is to make the `promtail` user part of the 
+same group as the user used to run the EVM node.
+
+### Adding log sinks
+
+The EVM Node uses the same [logging features](https://tezos.gitlab.io/user/logging.html#file-descriptor-sinks)
+as the [octez node](https://tezos.gitlab.io/). It is possible to add new 
+`sinks` using [environment variables](https://tezos.gitlab.io/user/logging.html#environment-variables).
+
+For example, the following definition will add logs in json format, with a 
+daily rotation, 4 file retention, with `rw-r-----` unix rights, in the 
+(fictional) `/some/data/dir/json_daily_logs` directory.
+
+```
+TEZOS_EVENTS_CONFIG="file-descriptor-path:///some/data/dir/json_daily_logs/daily.log?daily-logs=4&format=one-per-line&create-dirs=true&chmod=0o640"
+```
+
+This will result in files containing one json object per line.
+```
+...
+{"fd-sink-item.v0":{"hostname":"etherlink-pinata-sequencer","time_stamp":1730505587.606614,"section":["evm_node","dev"],"event":{"blueprint_injection.v0":"505717"}}}
+{"fd-sink-item.v0":{"hostname":"etherlink-pinata-sequencer","time_stamp":1730505590.115897,"section":["evm_node","dev"],"event":{"evm_events_follower_upstream_blueprint_applied.v0":{"level":"505714","hash":"0xfb6f61fe268481f1d18d31d7567011065418379b34722e83c517a6c509d3dfdb"}}}}
+...
+```
+
+Such files can be scraped using the following Promtail configuration:
```

Comment:
```suggestion
Such files can be scraped using the following Promtail configuration.
This will create a json event containing only the `event` of each log line.
```